src/containers/email-verification/EmailVerification.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Modal from '@mui/material/Modal'
+import IconButton from '@mui/material/IconButton'
+import CloseIcon from '@mui/icons-material/Close'
+import { styles } from './EmailVerification.styles'
+
+interface Props {
+  open: boolean,
+  handleClose: () => void
+}
+const EmailVerification: React.FC<Props> = ({ open, handleClose }) => {
+  return (
+    <Modal
+      aria-describedby='confirmation-modal-description'
+      aria-labelledby='confirmation-modal-title'
+      componentsProps={{
+        backdrop: {
+          style: {
+            backgroundColor: 'rgba(38, 50, 56, 0.75)'
+          }
+        }
+      }}
+      onClose={handleClose}
+      open={open}
+    >
+      <Box sx={styles}>
+        <IconButton
+          aria-label='close'
+          onClick={handleClose}
+          sx={{ position: 'absolute', right: 8, top: 8 }}
+        >
+          <CloseIcon />
+        </IconButton>
+        <Box
+          component='img'
+          src='/src/assets/img/email-needs-verified/email-needs-verified.svg'
```

Comment:
Instead of using the /src/assets/img/email-needs-verified/email-needs-verified.svg path, it might be better to use the image import directly in the component to avoid path problems and make the component easier to maintain
Maybe it's not so nessessory, but I saw it's a good practice in our project 

``` 
import EmailVereficationImage from '~/assets/img/email-needs-verified/email-needs-verified.svg'
```
```suggestion
        <Box
          component='img'
          src='EmailVereficationImage'
```